keps/sig-release/2572-release-cadence/README.md

```diff
@@ -270,7 +270,7 @@ This KEP will be in the `alpha` stage for the Kubernetes 1.22 Release. During
 this time, SIG-Release will focus on communication of the cadence change through
```

Comment:
@divya-mohan0209 -- if you're up for making the corrections throughout, yes please!

Per the style guide, SIG names are not referenced in hyphenated form: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#language

Reply:
Done, please review :)

Reply:
Also, I rebased it to avoid multiple commits + force merged it like we do in docs. Did not realize that this would throw up slackbot messages. Sorry about that!